### PR TITLE
Provision blob storage

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[Provision new BLOB](https://tools.hmcts.net/jira/browse/BPS-805)

### Change description ###

Provisioning new blob storage for `reform-scan` product:

- including palo alto
- storing secrets
- creating containers

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/palo-alto.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+locals {
+  trusted_vnet_name           = "core-infra-vnet-${var.env}"
+  trusted_vnet_resource_group = "core-infra-${var.env}"
+  trusted_vnet_subnet_name    = "palo-trusted"
+}
+
+module "palo_alto" {
```
Comment (timja) [inline infrastructure/palo-alto.tf]: I would be quite wary of having the palo alto infrastructure in an application repository,
the automation is quite poor, runs on every build and sometimes fails

you probably want the infrastructure provisioned in a separate  repo just for infrastructure?

Comment (doncem) [reply infrastructure/palo-alto.tf]: ah. so here comes the culprit which requires separate infra. will it be safe to move out all the rest as well? good thing nothing is used atm

Comment (nitinprabhu) [reply infrastructure/palo-alto.tf]: Agree with @timja else we would getting blocked with every build in future

Comment (doncem) [reply infrastructure/palo-alto.tf]: ok. will extract what's current and raise this in new repo

Comment (timja) [reply infrastructure/palo-alto.tf]: it's this guy + the DNS challenges required  for a custom domain on the storage account can be quite a pain...

**Comment by doncem** [comment]
will move to new repo

**Pull request closed without merging**